pkg/api/convertertoagentpoolonlyapi_test.go

```diff
@@ -54,19 +54,19 @@ func TestConvertV20180331AgentPoolOnlyOrchestratorProfile(t *testing.T) {
 		t.Error("error in orchestrator profile kubernetesVersion conversion")
 	}
 
-	if api.KubernetesConfig.NetworkPolicy != "" {
+	if api.KubernetesConfig.NetworkPolicy != "none" {
 		t.Error("error in orchestrator profile networkPlugin conversion")
 	}
 
-	if api.KubernetesConfig.ServiceCIDR != "" {
+	if api.KubernetesConfig.ServiceCIDR != "10.0.0.0/16" {
```

Comment:
can you define consts for `10.0.0.0/16` and `172.17.0.1/16`

Reply:
Sure, will do.